zigup.zig

```diff
@@ -223,15 +172,17 @@ fn help() void {
         \\Uncommon Usage:
         \\
         \\  zigup fetch-index             download and print the download index json
+        \\  zigup show-config             print the current configuration in ZON format
```

Comment:
Same thing here, maybe this new command is good but I'm not sure yet.  NULL hypothesis is to leave it out for now.

Reply:
Okay I take it back, you're probably right. Removed and resolved.